cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -174,6 +174,7 @@ func create(ctx context.Context) (err error) {
 		genOptions := []generate.GenOption{
 			generate.WithInstallImage(nodeInstallImage),
 			generate.WithDebug(configDebug),
+			generate.WithPersist(false),
```

Comment:
should we use defaults here? as otherwise we're not testing most common (default case)

it might make sense to disable config persistence for upgrade tests when going from 0.3/0.4 which doesn't do config persist by default

Reply:
What do you think about [this](https://github.com/talos-systems/talos/pull/2024/files#diff-4f6e60bfe3f88bb8b5699b02b63e29d0R179)? I wanted the docker creation behavior to not persist since there's no use in doing so. But this will also allow the default case to be tested w/ firecracker.

Reply:
sgtm, I just left comment how to make it more generic via other call :+1: